rllib/core/learner/learner.py

```diff
@@ -1054,30 +1070,58 @@ def additional_update(
 
         Example:
 
-        .. code-block:: python
-
-            class DQNLearner(TorchLearner):
-
-                def additional_update_for_module(self, module_id: ModuleID, tau: float):
-                    # perform polyak averaging update
-                    main = self.module[module_id].main
-                    target = self.module[module_id].target
-                    for param, target_param in zip(
-                        main.parameters(), target.parameters()
-                    ):
-                        target_param.data.copy_(
-                            tau * param.data + (1.0 - tau) * target_param.data
-                        )
-
-        And inside a training loop:
-
-        .. code-block:: python
-
-            for _ in range(100):
-                sample = ...
-                self.learner.update(sample)
-                if self.learner.global_step % 10 == 0:
-                    self.learner.additional_update(tau=0.01)
+        .. doctest::
+
+            >>> from ray.rllib.algorithms.ppo.torch.ppo_torch_rl_module import (
+            ...     PPOTorchRLModule
+            ... )
+            >>> from ray.rllib.algorithms.ppo.ppo_catalog import PPOCatalog
+            >>> from ray.rllib.algorithms.ppo.torch.ppo_torch_learner import (
+            ...     PPOTorchLearner
+            ... )
+            >>> import gymnasium as gym
+
+            >>> env = gym.make("CartPole-v1")
+
+            Create a single agent RL module spec.
```

Comment:
Need to confirm, but I don't think we can have plaintext like this in the `.. doctest` block. You might need to unindent this and do multiple doctest

Reply:
@bveeramani From executing the doctests locally, this does not complain